gui/src/components/Onboarding/index.tsx

```diff
@@ -29,18 +29,21 @@ export function Onboarding() {
   const step = steps[activeStep];
 
   return (
-    <Container disableGutters maxWidth="sm" sx={{ mt: 8, mb: 10, px: 3 }}>
-      <Stack alignItems="center">
-        <step.component onSubmit={handleNext} />
-
-        <MobileStepper
-          steps={steps.length}
-          position="static"
-          activeStep={activeStep}
-          nextButton={<></>}
-          backButton={<></>}
-        />
-      </Stack>
-    </Container>
+    <>
+      <Box sx={{ height: 64 }} data-tauri-drag-region="true"></Box>
```

Comment:
You have this in the navbar as well. The `data-tauri-drag-region` makes it so this element can be "dragged" to position the window.

Reply:
sure, but in the navbar it's inside the `Toolbar`. here seems to be a randomly placed 64-height box, with no context.

- is this taking up height in the view? why not put a Toolbar here too then?
- does this mean we can only drag when clicking this element?
- does the same thing happen in dialogs? if so this should probably be made as more general solution for the whole App

Reply:
Actually, that's a great point. The same applies to dialogs, so maybe I can just make a component to handle this stuff?

Reply:
Ok, I added a DraggableToolbar that handles this. The dialogs don't need it because they are created with a titlebar, unlike the main window. This is ok for now.